plugins/woocommerce/client/admin/client/settings-payments/components/modals/woo-payments-reset-account-modal.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Button, Modal } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './modals.scss';
+import { getWooPaymentsResetAccountLink } from '~/settings-payments/utils';
+
+interface WooPaymentsResetAccountModalProps {
+	isOpen: boolean;
+	onClose: () => void;
+}
+
+export const WooPaymentsResetAccountModal = ( {
+	isOpen,
+	onClose,
+}: WooPaymentsResetAccountModalProps ) => {
+	const [ isResettingAccount, setIsResettingAccount ] = useState( false );
+
+	const handleResetAccount = () => {
+		setIsResettingAccount( true );
+
+		window.location.href = getWooPaymentsResetAccountLink();
+	};
+
+	return (
+		<>
+			{ isOpen && (
+				<Modal
+					title={ __( 'Reset your test account', 'woocommerce' ) }
+					className="woocommerce-woopayments-modal"
+					isDismissible={ true }
+					onRequestClose={ onClose }
+				>
+					<div className="woocommerce-woopayments-modal__content">
+						<div className="woocommerce-woopayments-modal__content__item">
+							<div>
+								<span>
+									{ __(
+										'When you reset your test account, all data — including your WooPayments account details, test transactions, and payouts history — will be lost. This action cannot be undone, but you can create a new test account at any time.',
```

Comment:
Let's use `sprintf` and extract `WooPayments` from the translatable string.

Reply:
will do!

Reply:
thanks! fixed in https://github.com/woocommerce/woocommerce/pull/53590/commits/4c477eed54c5a1ad31ae576cddbb0c8f1743ecd9